db/migrate/20160719142419_create_projects.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProjects < ActiveRecord::Migration[5.0]
+  def change
+    create_table :projects do |t|
+      t.integer :project_id
```

Comment:
I think it will be better to just use `id` instead of `project_id`. If you remove `project_id` and leave the rest, the column `id` will be added by rails automatically when applying this migration.